monkey/monkey_island/cc/next_ui/package.json

```diff
@@ -4,9 +4,15 @@
   "private": true,
   "scripts": {
     "dev": "node server-dev.js",
+    "prod": "SSL_CERT_PATH=../server.crt SSL_KEY_PATH=../server.key NEXT_PUBLIC_JAVASCRIPT_RUNTIME_PORT=8443 node server-prod.js",
     "build": "node build-standalone.js",
     "lint": "next lint",
-    "test": "jest"
+    "test": "jest",
+    "prod:e2e": "npm run build && npm run prod",
+    "e2e": "START_SERVER_AND_TEST_INSECURE=1 start-server-and-test prod:e2e https://localhost:8443 cypress:run_e2e",
```

Comment:
`START_SERVER_AND_TEST_INSECURE=1` won't work on windows. Maybe lets use [cross-env](https://www.npmjs.com/package/cross-env)

Reply:
What's `start-server-and-test`?